taxi/tests/test_views.py

```diff
@@ -0,0 +1,180 @@
+from django.test import TestCase
+from django.urls import reverse
+
+from taxi.models import Driver, Manufacturer, Car
+
+
+class PublicTests(TestCase):
+    def setUp(self) -> None:
+        manufacturer = Manufacturer.objects.create(
+            name="test",
+            country="test country"
+        )
+        Driver.objects.create(
+            username="test",
+            password="test12345"
```

Comment:
The `password` field should be hashed when creating a `Driver` instance. Consider using Django's `set_password` method to ensure the password is stored securely.